### PR TITLE
Update past-exec.md and exec.md; Create exec-W16.md; Fixes #166

### LINKED ISSUE #166: Update committee members for S16
See #59 for what that might look like, in terms of updating links, archiving a copy of the previous committee page, etc.

### PR DESCRIPTION
**Pull request by techiechelle**

### CODE REVIEW

**Comment by evykassirer** [comment]
Looking sweet! Nice work figuring out all the file changes. 
You can see what the new website looks like at https://wics-site-pr-204.herokuapp.com/our-volunteers/
The last step is to squish all your commits together so there's only one commit in this pull request.

This is something that was confusing to me for a while, so I'm gonna explain it here just in case (sorry if you already know how to do it)
Docs about the command you use: https://help.github.com/articles/about-git-rebase/

First do `git rebase -i HEAD~8` to say you want to squish all the commits since 8 behind your current position
Then you will see a list of all your commits. Keep the first one as `pick` ___, and then choose `fixup` (or you can just type `f`) for the rest of the commits. Save and close, and there should just be one commit there! 🎉  (you can do `git log` to confirm that)

When you push that change, you'll have to do `git push --force` because git will be confused and think you're missing all the commits (when really you just turned them all into a new single commit)

**Comment by ehashman** [comment]
There is a typo: this term should have "Big CSters Director", not plural.

**Comment by techiechelle** [comment]
I made a commit to fix the typo

**Comment by fboxwala** [comment]
You will still need to squash your commits (as evy described above) before we can merge

File: content/pages/exec.md [hunk 1]
```diff
@@ -11,38 +11,29 @@ You can view [past terms' committees here]({filename}/pages/past-exec.md).
...
-Anna Lorimer (`aemhlori`)
+Evy Kassirer (`eawkassi`)
 
 ## Director of Finance ##
 
-Aurora Huang (`c36huang`)
+Zarina Mamoukhouva (`zrmamouk`)
 
-## Big CSters Directors ##
+## Big CSter Directors ##
```
Comment (ehashman) [inline content/pages/exec.md]: This should be `Big CSters Director` not Director**s**.

**Comment by fboxwala** [comment]
lgtm!

**Pull request merged**